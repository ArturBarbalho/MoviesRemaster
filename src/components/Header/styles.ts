import styled from 'styled-components'
export const Wrapper = styled.main`
display:flex;
align-items: center;
background-color:rgb(0,0,0,0.5);
position:absolute;
z-index:1;
width:100%;
justify-content:space-around;
a{
    text-decoration:none;
    font-size:larger;
    color: white;
}
`

export const Logo = styled.div`
img{
    width:70px;
    
}
h3{
color: white;
}
display:flex;
align-items:center;
`
export const DivLinks = styled.div `
display: flex;
justify-content:space-between;
width: 200px;
`