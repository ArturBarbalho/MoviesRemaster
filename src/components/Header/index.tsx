import Link from 'next/link'
import * as S from './styles'


const Header = () => {
  
  
return(
  <S.Wrapper>
     <S.Logo>
      <img src="movie-frame.png" alt="" />
      <h3>FREEPOP</h3>
    </S.Logo>    

     <S.DivLinks>
    <Link href='/'><a>MOVIES</a></Link>   
    <Link href='/tv'><a>TV SHOWS</a></Link>
     </S.DivLinks>
       
  </S.Wrapper>
)}

export default Header