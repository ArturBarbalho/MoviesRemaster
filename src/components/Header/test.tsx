import { render, screen } from '@testing-library/react'

import Header from '.'

describe('<Header />', () => {
  it('should render the heading', () => {
    const { container } = render(<Header />)

    expect(screen.getByRole('link', { name: /MOVIES/i })).toBeInTheDocument()
    expect(screen.getByRole('link', { name: /TV SHOWS/i })).toBeInTheDocument()
    expect(screen.getByRole('img')).toBeInTheDocument()

    expect(container.firstChild).toMatchSnapshot()
  })
})