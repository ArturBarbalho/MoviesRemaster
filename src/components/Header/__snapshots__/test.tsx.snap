// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Header /> should render the heading 1`] = `
<main
  class="styles__Wrapper-sc-u7lf7e-0 fFXhBd"
>
  <div
    class="styles__Logo-sc-u7lf7e-1 YqXdl"
  >
    <img
      alt=""
      src="movie-frame.png"
    />
    <h3>
      FREEPOP
    </h3>
  </div>
  <div
    class="styles__DivLinks-sc-u7lf7e-2 OxcRP"
  >
    <a
      href="/"
    >
      MOVIES
    </a>
    <a
      href="/tv"
    >
      TV SHOWS
    </a>
  </div>
</main>
`;
